### PR TITLE
Fix: refactoring and changes in homepage

### PR DESCRIPTION
**Pull request by sanu29**

### CODE REVIEW

File: src/component/AddressForm.jsx [hunk 1]
```diff
@@ -18,17 +18,24 @@ export function AddressForm(disp,setDisp) {
 
         {error !== "" ? <div className="alert alert-danger margin-none">{error}</div> : ""}
```
Comment (itsmebipul00) [inline src/component/AddressForm.jsx:19]: error && works well

File: src/component/AddressForm.jsx [hunk 2]
```diff
@@ -72,15 +79,20 @@ export function AddressForm(disp,setDisp) {
         <div className="address-form d-flex justify-content-evenly flex-wrap w-100 margin-top-1 ">
             <button className="btn btn-primary margin-none w-100"
                 onClick={() => {
-                    if (name === "" || Address === "" || pincode === "" || state === "") {
+                    const pattern = `[7-9][0-9]{9}` 
+                    if (name === "" || Address === "" || pincode === "" || state === ""||mobileNumber==="") {
```
Comment (itsmebipul00) [inline src/component/AddressForm.jsx:83]: Instead you can put required attribute in the input

**Review by itsmebipul00 — APPROVED** [review]
Good to merge Saniya!

File: src/pages/Auth/Login.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { LoginBody } from "../../component/LoginBody";
+import "./auth.css"
+export const Login = ()=>{
+  return(  <>
+
+       
+        <LoginBody/>
+      
+    </>)
```
Comment (iRohitGaur) [inline src/pages/Auth/Login.jsx:9]: suggestion: unless you need this file for additional jsx, better to use `<LoginBody />` directly after renaming it to `<Login />`

File: src/pages/Auth/signup.jsx [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { SignupBody } from "../../component/SignupBody"
+
+export const Signup = ()=>{
+   return( <>
+        <SignupBody/>
+   
+    </>)
```
Comment (iRohitGaur) [inline src/pages/Auth/signup.jsx:7]: suggestion: unless you need this file for additional jsx, better to use `<SignupBody />` directly after renaming it to `<Signup />`

File: src/pages/Order/Order.jsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+export const Order = () =>{
+
+    return(
+        <>
+    <Header/>
+    <div className="profile-contianer">       
+    <ProfileSideBar/>
+    <OrderBody/>
+      </div>
```
Comment (iRohitGaur) [inline src/pages/Order/Order.jsx:15]: suggestion: alignment issue. use a formatter

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Pull request merged**